app/css/climbingRoutes.css

```diff
@@ -1,7 +1,7 @@
 .allRoutes {
```

Comment:
![Screen Shot 2020-03-04 at 3 11 45 PM](https://user-images.githubusercontent.com/53950748/75920575-a996e800-5e2d-11ea-8c15-9b08323be9e6.png)

The 4th card is a small issue when changing the browser's width. Maybe consider setting the maximum width of each card.

Reply:
Done